common/lib/xmodule/xmodule/course_module.py

```diff
@@ -740,6 +740,16 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    enable_unproctored_exams = Boolean(
+        display_name=_("Enable Unproctored Exams"),
```

Comment:
@andy-armstrong I like Rob's suggestion, tweaked it to:
"Allow Opting Out of Proctored Exams"